disk/disk_linux.go

```diff
@@ -486,7 +486,7 @@ func IOCountersWithContext(ctx context.Context, names ...string) (map[string]IOC
 	return ret, nil
 }
 
-func udevData(ctx context.Context, major uint32, minor uint32, name string) (string, error) {
+func udevData(ctx context.Context, major, minor uint32, name string) (string, error) {
```

Comment:
I know such changes are implied by gofumpt "extra" rule, is this one linked to a go-critic rule?